setup.py

```diff
@@ -6,7 +6,7 @@
     {
         "pytest~=6.0",
         "pytest-cov>=2.5.1",
-        "scikit-hep-testdata>=0.4.0",
+        "scikit-hep-testdata>=0.4.36",
```

Comment:
Are there files in `scikit-hep-testdata` `v0.4.36` that are required here that aren't in previous versions?

Reply:
Ah, yes indeed looking at https://github.com/scikit-hep/scikit-hep-testdata/releases/tag/v0.4.36. Never mind.